4/Dockerfile

```diff
@@ -40,7 +40,7 @@ RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
   && rm "node-v$NODE_VERSION-linux-$ARCH.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt \
   && ln -s /usr/local/bin/node /usr/local/bin/nodejs
 
-ENV YARN_VERSION 1.3.2
+ENV YARN_VERSION 1.5.1
```

Comment:
I am not sure whether this change is wanted in this context (PR). Are you sure about this?

Reply:
I don't know, that come from `./update.sh`. I shouldnt have run that?

Reply:
fixed